08-transport.Rmd

```diff
@@ -28,36 +30,41 @@ It is therefore unsurprising that transport researchers have long turned to geoc
 This chapter provides an introduction to geographic analysis of transport systems.
 We will explore how movement patterns can be understood at multiple geographic levels, including:
 
-- **Areal units**: transport can be understood simply in terms of zonal aggregates such as the main mode and average distance of trips made people living in a particular zone.
+<!-- What is the main mode? I don't know the term. -->
+- **Areal units**: transport can be understood simply in terms of zonal aggregates such as the main mode and average distance of trips made by people living in a particular zone.
 - **Desire lines**: straight lines that represent 'origin-destination' data that records how many people travel (or could travel) between places (points or zones) in geographic space.
 - **Routes**: these are circuitous (non-straight) routes, typically representing the 'optimal' path along the route network between origins and destinations along the desire lines defined in the previous bullet point.
-- **Nodes**: these are points in the transport system that can represent common origins and destinations (e.g. with one centroid per zone) and public transport stations such as bus stops and rail stations.
+- **Nodes**: these are points in the transport system that can represent common origins and destinations (e.g., with one centroid per zone) and public transport stations such as bus stops and rail stations.
 - **Route networks**: these represent the system of roads, paths and other linear features in an area. They can be represented as purely geographic entities or as a graph.
+<!-- Not sure but maybe a word is missing, to me values representing 'flow' sounds a bit awkward (but then again I am not a native speaker --> 
 Their features are segments which can be assigned values representing 'flow', the number of people expected to use a particular street or path.
 
 The lowest but hardest-to-model level in transport systems is **agents**, mobile entities like you and me.
-Some models built using software such as [MATSim](http://www.matsim.org/) can effectively capture the dynamics of transport systems using an agent-based modelling (ABM) approach at high spatial and temporal resolution.
+Some models can effectively capture the dynamics of transport systems using an agent-based modelling (ABM) approach at high spatial and temporal resolution, e.g., [MATSim models](http://www.matsim.org/).
+<!-- is that true? More inaccessible than geocomputation or geostats with R? -->
 These tend to be complex and computationally demanding [@horni_multi-agent_2016], meaning they are inaccessible to most people.
```

Comment:
Is that true? More inaccessible than geocomputation or geostats with R?

Reply:
I believe so. Hopefully that will change and I remember having second thoughts about that statement. I plan to tone-it-down (or feel free to tone-down yourself in this PR if you think of an appropriate way of saying 'full agent based transport models running at high temporal resolution are complex compared with aggregate level models' here)